frontend/src/admin/components/sidebar/user-list/user-list.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from "react";
+import Users from "../../../users.json";
+import {User} from "../../../models/user";
+import UserListItem from "./user-list-item";
+import IconButton from "./icon-button";
+import "./user-list.scss";
+
+export default function UserList() {
+    const [selectedUser, setSelectedUser] = useState<User>();
+    return (
+        <section className="c-user-list">
+            <div className="c-user-list__header">
+                <h2>Users</h2>
+                <IconButton/>
+            </div>
+            <div className="c-user-list__items">
+                {Users.users.map((user) => (
+                    <UserListItem
+                        user={user}
```

Comment:
You parse a user here, but your `UserListItem` does not take a user in its properties...

Also, five lines down you parse `user.name` to the label property, but since you "are parsing" the user, you don't need to parse `user.name` as well, since that info is allready in the User.
So instead of exposing a `label` property, just parse the use User and inside `UserListItem` you just set `user.name` directly where you want to display it